README.md

```diff
@@ -1,61 +1,93 @@
 # OpenMRS ESM Patient Chart
 
-## What is this?
+The `openmrs-esm-patient-chart` is a frontend module for the OpenMRS SPA. It bundles together various microfrontends that constitute widgets in a patient dashboard. These widgets include:
+
+- [Allergies](packages/esm-patient-allergies-app/README.md)
+- [Appointments](packages/esm-patient-appointments-app/README.md)
+- [Attachments](packages/esm-patient-attachments-app/README.md)
+- [Biometrics](packages/esm-patient-biometrics-app/README.md)
+- [Conditions](packages/esm-patient-conditions-app/README.md)
+- [Forms](packages/esm-patient-forms-app/README.md)
+- [Immunizations](packages/esm-patient-immunizations-app/README.md)
+- [Medications](packages/esm-patient-medications-app/README.md)
+- [Notes](packages/esm-patient-notes-app/README.md)
+- [Patient banner](packages/esm-patient-banner-app/README.md)
+- [Patient chart](packages/esm-patient-chart-app/README.md)
+- [Programs](packages/esm-patient-programs-app/README.md)
+- [Test results](packages/esm-patient-test-results-app/README.md)
+- [Vitals](packages/esm-patient-vitals-app/README.md)
 
-openmrs-esm-patient-chart is the patient chart frontend module and all its widgets. This creates a patient dashboard microfrontend for the OpenMRS SPA. It provides a simple dashboard with cards detailing the
-patient's information, such as vitals, demographic and relationships.
+In addition to these widgets, two other microfrontends exist that encapsulate cross-cutting concerns. These are:
+
+- [Common lib](packages/esm-patient-common-lib/README.md)
+- [Patient chart](packages/esm-patient-chart-app/README.md)
 
 ## Setup
 
-See the guidance in the [Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/getting_started/setup?id=developing-microfrontends-in-a-lerna-monorepo).
-This repository uses Yarn and Lerna.
+Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
+
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna). 
 
-To start the dev server for a specific package, run
+To start a dev server for a specific microfrontend, run:
 
 ```bash
-yarn start --sources 'packages/esm-patient-<package>-app'
+yarn start --sources 'packages/esm-patient-<insert-package-name>-app'
 ```
 
-This will start a dev server for that package as well as `esm-patient-chart-app`.
+This command uses the `openmrs` tooling to fire up a dev server on the default port (`8081`) for the specified microfrontend in addition to `esm-patient-chart-app`.
 
-To start a dev server running all the packages, run
+To start a dev server running all the packages, run:
 
 ```bash
 yarn start-all
 ```
 
-Note that this is extremely resource-intensive.
+Note that this is very resource-intensive. 
+
+The recommended approach for working on multiple microfrontends is to run each one separately on a different port and then use [import map overrides](http://o3-dev.docs.openmrs.org/#/getting_started/setup?id=import-map-overrides). 
```

Comment:
I always just use multiple `--sources` arguments. We should probably mention that you can do that.

Reply:
Oh, how so?

Reply:
e.g. `yarn start --sources packages/esm-patient-allergies-app/ --sources packages/esm-patient-medications-app/`